contracts/CO30_Munson_Deadend.tem_vinjesvingenc/edit_me/briefing/objectives.html

```diff
@@ -0,0 +1,7 @@
+<font color='#E3D310'>PRIMARY</font>
+Recapture the FOB and destroy\capture the T-100 tank.
```

Comment:
```suggestion
- Recapture the FOB
- Destroy / Capture the T-100 tank
```
The objectives should always be a list of items their smallest form